autopilot/migrator.go

```diff
@@ -217,7 +217,6 @@ OUTER:
 		if err := b.RefreshHealth(m.ap.shutdownCtx); err != nil {
 			m.ap.RegisterAlert(m.ap.shutdownCtx, newRefreshHealthFailedAlert(err))
 			m.logger.Errorf("failed to recompute cached health before migration: %v", err)
-			return
```

Comment:
Info logging `recomputed slab health in x seconds` needs to move into an `else` if we remove the `return` here.